editions/tw5.com/tiddlers/nodejs/Installing a plugin - client server.tid

```diff
@@ -0,0 +1,38 @@
+created: 20220611123344385
+modified: 20220612091241651
+tags: [[TiddlyWiki on Node.js]] PluginsCS
+title: Installing a plugin - client server
+
+Follow these instructions when using TiddlyWiki under client-server (Node.js) configuration:
+
+# Identify the plugins you want to install using the ''Plugins'' tab of [[control panel|$:/ControlPanel]] (''don't'' install the plugins from here, though). Plugins are identified by their type (ie language, theme or plugin) and their publisher and title. For example, the plugin `$:/plugins/tiddlywiki/internals` is referred to as ''tiddlywiki/internals''
+
+# Quit the server if it is running
+
+# Edit the `tiddlywiki.info` file (it is in JSON format) and locate the `plugins`, `themes` or `languages` section (see below)
+
+# Add entries corresponding to the plugins you wish to add. 
+#* Take care to retain ''commas ''to separate items, 
+#* Do ''not'' terminate the last item in a list with a comma
+
+# Restart the server
+
+```
+{
+	"plugins": [
+		"tiddlywiki/codemirror"
+	],
+	"themes": [
+		"tiddlywiki/vanilla",
+		"tiddlywiki/snowwhite"
+	],
+    "languages": [
+        "es-ES",
+        "fr-FR",
+        "en-EN"
+    ]
+}
+```
+
+<$macrocall $name=".note" _="An overview, how to handle plugins can be found at: [[Plugins]].<br>Also see: [[Installing custom plugins on Node.js]]."/>
```

Comment:
The comma here is not needed in English. Clearer to say "An overview of working with plugins can be found at [[Plugins]]<br>Also see: [[Installing custom plugins on Node.js]]."/>`